src/scss/components/_tabs.scss

```diff
@@ -0,0 +1,46 @@
+.nav-tabs {
+  border-bottom: none; 
+
+  .nav-item {
+    .nav-link {
+      border: $nav-tabs-border-width solid $nav-tabs-border-color;
+      border-radius: 0;
+      border-right: 0;
+      padding: $nav-tabs-link-padding-y $nav-tabs-link-padding-x;
+
+      @include media-breakpoint-up(lg) {
+        @include font-size($h4-font-size-lg);        
+      }
+      &.active {
+        color: white;
+      }
+    }
+
+    &:first-child {
+      .nav-link {
+        border-radius: $nav-tabs-border-radius 0 0 $nav-tabs-border-radius;
+      }
+    }
+
+    &:last-child {
+      .nav-link {
+        border-radius: 0 $nav-tabs-border-radius $nav-tabs-border-radius 0;
+        border-right: $nav-tabs-border-width solid $nav-tabs-border-color;
+      } 
+    }
+  }
+}
+
+.nav-tabs-product {
+  max-width: calc-em(510);
+  .nav-item {
+    width: 33.33%;
+  }
+}
+
+.nav-tabs-real-results {
+  max-width: calc-em(410);
+  .nav-item {
+    width: 25%;
+  }
+}
```

Comment:
tabs should not have fixed width sizes. We just need to have each tab link at the same size